Sources/App/Migrations/076/UpdateRepositoryResetReadmes.swift

```diff
@@ -0,0 +1,33 @@
+// Copyright Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Fluent
+import SQLKit
+
+
+struct UpdateRepositoryResetReadmes: AsyncMigration {
+    func prepare(on database: Database) async throws {
+        guard let db = database as? SQLDatabase else {
+            fatalError("Database must be an SQLDatabase ('as? SQLDatabase' must succeed)")
+        }
+
+        try await db.raw("""
+        UPDATE repositories
+        SET s3_readme = jsonb_set(s3_readme, '{cached,githubEtag}', '""', FALSE);
```

Comment:
This is to reset the field, isn't it? Why aren't we just setting it to `NULL`? That way we don't "bake in" any format and can be sure it's readable.

Reply:
It's to reset just the `githubEtag` part of the JSON field. If we reset the field fully, we'll lose all README files for the duration of a full pass of ingestion as the URL to the S3 cached copy of the README is in the other part of this JSON field. 

I'd welcome verification of the SQL statement, though. I have tried it and it does what I expect it to. The `FALSE` at the end means that if there is no value for etag then it won't create it, which is also for safety around fields that do not yet have a value (for new packages yet to be ingested).

It seemed like the correct way to approach it to me.

Reply:
Oh, I see! That makes sense.

In that case we should put that in a test though. We have a couple that do similar things, for example `test_pending_to_triggered_migration` and `test_DeleteArmBuilds_migration`.

Reply:
Added `test_migration076_updateRepositoryResetReadmes`